tests/Kerbero.Integration.Tests/Utilities.cs

```diff
@@ -0,0 +1,50 @@
+using System.Text.Json;
+using Kerbero.Domain.NukiAuthentication.Entities;
+using Kerbero.Infrastructure.Common.Context;
+
+namespace Kerbero.Integration.Tests;
+
+public class Utilities
+{
+	public Utilities()
+	{
+		var json = File.ReadAllText("JsonData/get-nuki-smartlock-response.json");
+		NukiJsonSmartlockResponse = JsonSerializer.Deserialize<dynamic>(json) ?? throw new InvalidOperationException();
+	}
+
+	public readonly object NukiJsonSmartlockResponse;
+	
+	#region Seed DB
+
+	public static void InitializeDbForTests(ApplicationDbContext db)
+	{
+		db.NukiAccounts.AddRange(GetSeedingMessages());
```

Comment:
```suggestion
		db.NukiAccounts.AddRange(GetSeedingMessages());
```
per il momento puo andare, pero in generale credo tu voglia seedare ogni volta in maniera diversa in base al test